classes/image_types_tegra.bbclass

```diff
@@ -332,7 +335,7 @@ copy_dtbs() {
         fi
         bbnote "Copying KERNEL_DEVICETREE entry $dtbf to $destination"
         cp -L "${DEPLOY_DIR_IMAGE}/$dtbf" $destination/$dtbf
-	if ${USE_UEFI_SIGNED_FILES}; then
+	if ${TEGRA_UEFI_USE_SIGNED_FILES} == "true"; then
```

Comment:
Here, in shell script, this could either be what you had before (`if ${TEGRA_UEFI_USE_SIGNED_FILES}`)  or it should be a valid POSIX shell construct (e.g., `if [ "${TEGRA_UEFI_SIGNED_FILES}" = "true" ]`).  The bare comparison with the double-equals isn't quite going to work.

Reply:
Well, that was rather sloppy of me.  I'll have a fix shortly.